### ISSUE TITLE
Unknown builtin op: torchvision::nms in LibTorch

### ISSUE THREAD

**Issue by DeriZSY** [pytorch/pytorch#57716]
## Unknown builtin op: torchvision::nms in LibTorch
pytorch vision: 1.8.0
libtorch version: 1.8.1

Try to trace the implementation of YOLOV5 [here](https://github.com/ultralytics/yolov5) and load it from LibTorch, which uses `torchvision.ops.nms` in the postprocessing step.

The model can be traced successfully, but error occurs when trying to load the traced model in LibTorch:
```
terminate called after throwing an instance of 'torch::jit::ErrorReport'
  what():
Unknown builtin op: torchvision::nms.
Could not find any similar ops to torchvision::nms. This op may not exist or may not be currently supported in TorchScript.
```
Are torchvision.ops supported in LibTorch at present? and what should I do to solve the problem?

**Comment by xsacha** [comment]
Same issue here when converting to torchscript:

Unknown builtin op: torchvision::deform_conv2d.
Could not find any similar ops to torchvision::deform_conv2d. This op may not exist or may not be currently supported in TorchScript.

Serialized   File "code/__torch__/torchvision/ops/deform_conv.py", line 16
    mask = torch.zeros([int(_2), 0], dtype=5, layout=None, device=torch.device("cuda:0"), pin_memory=False)
    bias = torch.zeros([_1], dtype=5, layout=None, device=torch.device("cuda:0"), pin_memory=False)
    input = ops.torchvision.deform_conv2d(x, _0, offset, mask, bias, 1, 1, 1, 1, 1, 1, 1, 1, False)
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
    return input

**Comment by gmagogsfm** [comment]
Torchvision needs to be installed separately for `nms` to be available, it doesn't come with `libtorch` automatically. Please install torchvision and try again:
https://github.com/pytorch/vision

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
